jep/211/README.adoc

```diff
@@ -350,6 +350,23 @@ Non-goals in the specification are defined to limit the scope of work.
 The main objective is to get Jenkins running with Java 11,
 there will be follow-up tasks to cleanup Illegal Reflective Access warnings and to adopt new features.
 
+=== Support of Java 10 and 12
```

Comment:
Possibly could be even widened to say something like basically we support the specified version, and we do *NOT* on any other/new version that we didn't explicitly say we support (I'm thinking about this to somehow cover upcoming JDK 13, 14...)

Reply:
IMHO it's an overkill for this JEP.
I would rather cover it in a new JEP for non-LTS Java versions once we start setting up a process for it

Reply:
```suggestion
=== Support of Java 10 and 12+
```